### ISSUE TITLE
测试运行完显示内容，没有出现测试集结果

### ISSUE THREAD

**Issue by Jessie071115** [Determined22/zh-NER-TF#51]
vocab_size: 3905
Namespace(CRF=True, batch_size=64, clip=5.0, demo_model='1521112368', dropout=0.5, embedding_dim=300, epoch=40, hidden_dim=300, lr=0.001, mode='test', optimizer='Adam', pretrain_embedding='random', shuffle=True, test_data='data_path', train_data='data_path', update_embedding=True)
.\data_path_save\1521112368\checkpoints/model-31680
C:\Jessie\lib\site-packages\tensorflow\python\ops\gradients_impl.py:100: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
test data: 4631
=========== testing ===========
INFO:tensorflow:Restoring parameters from .\data_path_save\1521112368\checkpoints/model-31680
Restoring parameters from .\data_path_save\1521112368\checkpoints/model-31680
